src/utils/colonyActions.ts

```diff
@@ -219,12 +224,14 @@ export const getSafeTransactionActionType = (
   safeTransactions: SafeTransaction[],
 ) => {
   if (
-    actionType === ColonyExtendedActions.SafeTransactionInitiated &&
+    (actionType === ColonyExtendedActions.SafeTransactionInitiated ||
+      actionType === AddedMotions.SafeTransactionInitiatedMotion) &&
     safeTransactions
   ) {
     if ((safeTransactions || []).length >= 2) {
       return TransactionTypes.MULTIPLE_TRANSACTIONS;
     }
+
     const type = safeTransactions[0].transactionType;
```

Comment:
Just need the `?` here to capture the undefined case. Maybe add a comment two about why it's coming through as undefined and update the typing?

Reply:
Hmm, actually I take that back. I just get what appears to be a fallback and have to refresh to get the actual motion screen: 

![fallback`](https://user-images.githubusercontent.com/64402732/199744542-9db63d95-8389-4089-a327-49676024171b.png)

Reply:
Can you check that your branch is up-to-date? In your screenshot it seems like the subgraph didn't return the IPFS data of the transaction and that's something that it should be fixed by my latest changes